src/Module.ts

```diff
@@ -907,6 +896,19 @@ export default class Module {
 		}
 	}
 
+	private includeAndGetReexportedExternalNamespaces(): ExternalVariable[] {
```

Comment:
I found it a little more straightforward to not push all external namespaces individually to the namespace variable but rather have the context function take care of all that.